Makefile

```diff
@@ -316,7 +316,7 @@ else
 endif
 
 .PHONY: ci-manifest-build
-ci-manifest-build: manifest-build ## Build CI manifest
+ci-manifest-build: manifest ## Build CI manifest
```

Comment:
if "ci-manifest-build" runs "manifest" which itself does a push, then "ci-manifest-push" is now useless (or overlapping)?

Reply:
but it works like this in other components:
- `make images` build and push manifest
- `make ci-manifest` overrides the manifest shortlived